packages/core/src/JWEditor.ts

```diff
@@ -186,6 +172,37 @@ export class JWEditor {
         this._originalEditable.style.display = this.editable.style.display;
         this.el.remove();
     }
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Load a shortcut in the keymap depending on the platform.
+     *
+     * - If the shortuct has no platform property; load the shortuct in both platform ('mac' and
+     *   'pc').
+     * - If the shortuct has no platform property and the current platform is mac, modify the ctrl
+     *   key to meta key.
+     * - If the shortuct has a platform property, only load the shortcut for that platform.
+     * - If no `mapping.commandId` is declared, it means removing the shortcut.
```

Comment:
Well that confirms what I suspected about `removeShortcut`. It's never actually used, since removing a shortcut is still adding a shortcut anyway.

Reply:
remove `addShortuct` and `removeShortuct`, replace it with `bindShortcut`